README.md

```diff
@@ -43,21 +44,12 @@ Color::make('Field')->compact()
 Color::make('Field')->slider()
 ```
 
-and so on. You can use any one of: `chrome, compact, grayscale, material, photoshop, sketch, slider, swatches`.
-
-
-Palette support for adding custom colors to the pickers is also available:
-
-```
-Color::make('Field')->compact()->palette(['#beaf00', '#DEADAF', '#000']);
-```
-
+and so on. You can use any one of: `chrome, compact, grayscale, material, photoshop, sketch, slider, swatches`
 
 ## Credits
 
 -   [Timothy Asp](https://github.com/timothyasp)
 -   [Don Gilbert](https://github.com/dongilbert)
--   [marvinrabe](https://github.com/marvinrrabe)]
```

Comment:
Please unstage the changes to README.md.  Not sure why this is removed, and the documentation about `palette`

Reply:
Uhm.. not sure neither why this file was changed.. was a mistake I reverted it